content/en/opentelemetry/schema_semantics/semantic_mapping.md

```diff
@@ -105,6 +105,78 @@ Additional cloud provider-specific attributes are also mapped.
 | `url.full` | `http.url` |
 | `user_agent.original` | `http.useragent` |
 
+## Span Type mapping
+Datadog has a vendor-specific convention of "Span Type" represented by the `span.type` attribute.
+
+Based on the attributes included in your span, Datadog Agent and Datadog OpenTelemetry Components will attempt to infer the appropriate span type for better compatibility with other Datadog services. You may also explicitly set `span.type` attribute on any given span to override this logic using an [attributes][5] or [transform][6] processor, as well as by setting appropriate configuration values in OTel SDKs. 
+
+### OTel Span Attributes -> Datadog Span Type
+The following table shows the Span Type mapping logic that is used if feature flag `enable_receive_resource_spans_v2` is set. The chart is in order of precedence.
```

Comment:
It would be great to point out where the `enable_receive_resource_spans_v2` needs to be anabled.
Is that on the OTel Collector?